routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
+
+    attr_reader :subscriber, :callback, :topics, :events
+
+    def initialize(options)
+      @subscriber = options.fetch('subscriber')
+      @callback   = options.fetch('callback')
+      @topics     = options.fetch('topics')
+      @events     = options.fetch('events').symbolize_keys
+    end
+
+    def attributes
```

Comment:
Please — I'm wary of adding features we don't really know are required/requested/useful.